src/styles/blocks/card.scss

```diff
@@ -0,0 +1,69 @@
+
+.card {
+  margin-right: 48px;
+  margin-bottom: 48px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+      
+  &__image {
+    box-sizing: border-box;
+    margin-bottom: 40px;
+  }
+
+  &__product-name {
+    font-size: 12px;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    font-weight: 400;
+    font-size: 10px;
+    margin-bottom: 16px;
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    font-weight: 400;
+    font-size: 10px;
+    margin-bottom: 24px;
+    align-items: center;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+  }
+
+  &__price-word {
+    align-self: center;
+    font-weight: 400;
+    font-size: 12px;
+  }
+
+  &__buy-button {
+    box-sizing: border-box;
+    text-align: center;
+    line-height: 40px;
+    font-size: 14px;
+    text-transform: uppercase;
+    height: 40px;
+    border: 1px solid ;
+    border-radius: 5px;
+    background-color: #00acdc;
+    text-decoration: none;
+    color: #fff;
+  }
+
+  &__buy-button:hover {
+    color: #00acdc;
+    background-color: #fff;
+    border-color: #00acdc;
+  }
```

Comment:
use only background-color in :hover

Reply:
this solution has been approved by mentors